### PR TITLE
Add Echo to Ruby

### PR DESCRIPTION
**Pull request by afman42**
Implementation
- string
  - [x] echo

unit test
- string
  - [x] echo

### CODE REVIEW

**Review by ronnygunawan — CHANGES_REQUESTED** [review]
Testnya belum ngetest apa-apa. Mungkin bisa pake cara ini: https://stackoverflow.com/questions/16507067/testing-stdout-output-in-rspec

File: ruby/spec/pehape/string/echo_spec.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+RSpec.describe Pehape do
+  it "echo all type data" do
+    expect(Pehape.echo("Pe ha pe")).to equal(nil)
```
Comment (elianiva) [inline ruby/spec/pehape/string/echo_spec.rb:5]: assert stdoutnya, jangan return valuenya
http://rspec.info/blog/2014/05/notable-changes-in-rspec-3/#new-output-matcher

**Review by elianiva — CHANGES_REQUESTED** [review]

**Comment by afman42** [comment]
oke om, makasih sarannya

**Comment by afman42** [comment]
bisa di cek output di command `bin/console` dan `Pehape.echo(blabla)`. om @ronnygunawan

**Review by ronnygunawan — APPROVED** [review]
Nice! LGTM

**Pull request merged**